packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.types.ts

```diff
@@ -114,4 +116,39 @@ export interface ICoachmarkTypes extends React.Props<Coachmark> {
    * Beacon color two.
    */
   beaconColorTwo?: string;
+
+  /**
+   * Text to announce to screen reader / narrator when Coachmark is displayed
+   */
+  ariaAlertText?: string;
+
+  /**
```

Comment:
@JasonGore Any suggestions here?  I'm not quite sure how make this interface look nicer.